install/operator/deploy/crds/syndesis_v1alpha1_syndesis_cr.yaml

```diff
@@ -23,6 +23,9 @@ spec:
     components:
         scheduled: true
         db:
+            registry: registry.redhat.io
```

Comment:
This would also need to go into `fuse-online-install`

Reply:
I think we have to change all this(registry is changed already) in the `conf/config.yaml` when building downstream. @orpiske 

`Images.PostgresqlImagePrefix centos` -> `Images.PostgresqlImagePrefix rhscl`
`Images.Support.Postgresql postgresql-95-centos7` -> `Images.Support.Postgresql postgresql-95-rhel7`

Reply:
So it doesn't need to be declared (or overwritten) in the CR?

Reply:
@lgarciaaco ACK.

Reply:
It can be overwritten using the CR, we for that kind of configuration we usually change the values in the config file at build time